main.go

```diff
@@ -36,12 +37,18 @@ func main() {
 			Description: "Gets specific config for service.",
 			Run: func(args []string) {
 				var name string
-				config := parseFlags("get", args, func(fs *flag.FlagSet) {
+				config, parseErr := parseFlags("get", args, false, func(fs *flag.FlagSet) []string {
+					required := []string{"name"}
+
 					fs.StringVar(&name, "name", "", "Name of the config item to retrieve.")
+
+					return required
 				})
 
+				check(config.Logger, parseErr, "", 2)
```

Comment:
What is the meaning of the exit code '2' here (i.e. why do we vary the code and not just use 1 everywhere)?

Reply:
This is an attempt to maintain [existing functionality](https://github.com/guardian/devx-config/pull/2/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261L166). My understanding was that the exit code 2 indicates "command line usage error".